src/components/ConfigScreen/ConfigScreen.tsx

```diff
@@ -40,6 +55,7 @@ export default class Config extends Component<ConfigProps, ConfigState> {
       isButtonLoading: false,
```

Comment:
nit: should this be `this.state = {...}:ConfigState`?

Reply:
Hmm when I try that I get `'ConfigState' only refers to a type, but is being used as a value here.ts(2693)`. Hovering over `this.state` also shows `(property) React.Component<ConfigProps, ConfigState, any>.state: Readonly<ConfigState>` so I'm not sure if this possible.

Reply:
Sorry I meant `this.state = {...} as ConfigState` but this shouldn't be a blocker.